modules/websocket/doc_classes/WebSocketClient.xml

```diff
@@ -25,7 +25,8 @@
 			</argument>
 			<description>
 				Connect to the given URL requesting one of the given [code]protocols[/code] as sub-protocol.
-				If [code]true[/code], is passed as [code]gd_mp_api[/code], the client will behave like a network peer for the [MultiplayerAPI]. Note: connections to non Godot servers will not work, and [signal data_received] will not be emitted when this option is true.
+				If [code]true[/code], is passed as [code]gd_mp_api[/code], the client will behave like a network peer for the [MultiplayerAPI], connections to non Godot servers will not work, and [signal data_received] will not be emitted.
```

Comment:
Looks like I did a mistake here when mass replacing "If true" by "If [code]true[/code]**,**", the comma is bogus. Same for the next line.

Reply:
Oh, it sounded strange, but my grade in grammar is not that good so I decided to leave it.
Fixed now.